modules/claims_api/app/controllers/claims_api/v1/application_controller.rb

```diff
@@ -193,17 +193,18 @@ def edipi_check
       end
 
       def validate_header_values_format
-        if header('X-VA-Birth-Date').blank?
+        errors = []
+        errors << 'birth date' if header('X-VA-Birth-Date').blank?
+        errors << 'first name' if header('X-VA-First-Name').blank?
+        errors << 'last name' if header('X-VA-Last-Name').blank?
```

Comment:
more nitpick but should we include the header value as it's directly inline with what they're "missing" 

so something like 
```
errors << 'X-VA-Birth-Date' if header('X-VA-Birth-Date').blank?
```

Reply:
Ya, I was not sure what they saw from their end, but that is an easy enough change to make here I will update it